### DISCUSSION TITLE
[Bug] scoop update scoop Updating Scoop... Updating 'extras' bucket... error: cannot open '.git/FETCH_HEAD': Permission denied

### DISCUSSION THREAD

**Discussion by DrVektor** [ScoopInstaller/Scoop#5905] [Help and Questions]
scoop update scoop
Updating Scoop...
Updating 'extras' bucket...
error: cannot open '.git/FETCH_HEAD': Permission denied
Updating 'main' bucket...
Updating 'nerd-fonts' bucket...
Updating 'nirsoft' bucket...
Updating 'nonportable' bucket...
Updating 'sysinternals' bucket...
Updating 'versions' bucket...
Scoop was updated successfully!

**Comment by rashil2000** [comment]
Try re-adding the extras bucket:
```
scoop bucket rm extras
scoop bucket add extras
```
